lua/plenary/path.lua

```diff
@@ -88,7 +88,7 @@ local function _normalize_path(filename)
           idx = idx + 1
       until idx > #parts
 
-      out_file = table.concat(parts, path.sep)
+      out_file = path.sep .. table.concat(parts, path.sep)
```

Comment:
Shouldn't this be `path.root` so it'll work on windows?

Reply:
probably did not know that existed, will fix